test/functional/device/motion-e2e-specs.js

```diff
@@ -1,61 +1,63 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { MOCHA_TIMEOUT, initSession, deleteSession } from '../helpers/session';
-import { SETTINGS_CAPS } from '../desired';
+import { SETTINGS_CAPS, amendCapabilities } from '../desired';
 
 
 const PREDICATE_SEARCH = '-ios predicate string';
```

Comment:
can we reuse this constant in other specs?

Reply:
Sure, I added a new file to helpers https://github.com/appium/appium-xcuitest-driver/commit/8babf26aec4efebce40d516333eeb30ad92bbceb